src/test/java/org/mule/module/apikit/deserializing/AttributeDeserializerTest.java

```diff
@@ -50,11 +49,11 @@ public class AttributeDeserializerTest {
   @Parameterized.Parameters(name = "Delimiter = {0} - {1}")
   public static Iterable<Object[]> data() throws Exception {
     ArrayHeaderDelimiter comma = COMMA;
-    ArrayHeaderDelimiter semicolon = createEnumValue(ArrayHeaderDelimiter.class, "SEMICOLON", 2, ";");
     return asList(new Object[][] {
         {comma, new ArrayHeaderAttributeDeserializer(comma)},
-        {comma, new DummyAttributeDeserializer()},
-        {semicolon, new ArrayHeaderAttributeDeserializer(semicolon)}
+        {comma, new DummyAttributeDeserializer()}
+        // {semicolon, new ArrayHeaderAttributeDeserializer(semicolon)} uncomment when semicolon value included in
+        // ArrayHeaderDelimiter
```

Comment:
```suggestion
```
I don't really understand why we were testing this. If the enum value does not exist then we should get rid of the commented code too.